it/src/main/java/org/corfudb/universe/universe/vm/VmConfigPropertiesLoader.java

```diff
@@ -1,46 +1,59 @@
 package org.corfudb.universe.universe.vm;
 
+import lombok.extern.slf4j.Slf4j;
 import org.corfudb.common.result.Result;
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
 import java.util.Properties;
 
 /**
  * Loads config data from properties files.
  */
+@Slf4j
 public class VmConfigPropertiesLoader {
 
     private VmConfigPropertiesLoader() {
         //prevent creating instances
     }
 
-    public static final String VM_PROPERTIES_FILE = "vm.properties";
-    public static final String VM_CREDENTIALS_PROPERTIES_FILE = "vm.credentials.properties";
+    public static final Path VM_PROPERTIES = Paths.get("vm.properties");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4663789)